ops/reth-cross-test/verification/l1_message_checker.py

```
@@ -0,0 +1,153 @@
+
+def verify_l1_message_inclusion(
+    sequencer_rpcs: dict[str, str],
+    l1_tx_hash: str,
+    timeout: int = 60,
+) -> dict:
+    """Wait for L1 deposit to appear in L2 blocks, verify inclusion rules.
+
+    Returns: {"pass": bool, "details": {...}}
+    """
+    # Wait for the L1 message to appear in an L2 block
+    first_rpc = list(sequencer_rpcs.values())[0]
```

Ruff (0.15.7)

[warning] Prefer `next(iter(sequencer_rpcs.values()))` over single element slice

Replace with `next(iter(sequencer_rpcs.values()))`

(RUF015)